modules/apps/commerce/commerce-cart-content-web/src/main/resources/META-INF/resources/cart/view.jsp

```diff
@@ -141,19 +141,45 @@ Map<Long, List<CommerceOrderValidatorResult>> commerceOrderValidatorResultMap =
 					%>
 
 					<liferay-ui:search-container-column-text
-						name="list-price"
+						name="price"
 					>
 						<c:if test="<%= commerceCartContentDisplayContext.hasViewPricePermission() %>">
 
 							<%
 							CommerceMoney unitPriceCommerceMoney = commerceOrderItem.getUnitPriceMoney();
+							CommerceMoney unitPromoPriceCommerceMoney = commerceOrderItem.getPromoPriceMoney();
 
 							if (commercePriceDisplayType.equals(CommercePricingConstants.TAX_INCLUDED_IN_PRICE)) {
 								unitPriceCommerceMoney = commerceOrderItem.getUnitPriceWithTaxAmountMoney();
+								unitPromoPriceCommerceMoney = commerceOrderItem.getPromoPriceWithTaxAmountMoney();
 							}
 							%>
 
-							<%= HtmlUtil.escape(unitPriceCommerceMoney.format(locale)) %>
+							<c:choose>
+								<c:when test="<%= !unitPromoPriceCommerceMoney.isEmpty() && CommerceBigDecimalUtil.gt(unitPromoPriceCommerceMoney.getPrice(), BigDecimal.ZERO) && CommerceBigDecimalUtil.lt(unitPromoPriceCommerceMoney.getPrice(), unitPriceCommerceMoney.getPrice()) %>">
```

Comment:
can you add this check in the display context, and call the method withe the business rule that is checking? It will make the cod 10000 more readable